web/src/features/charts/DataSources.tsx

```diff
@@ -53,6 +88,19 @@ export function DataSources({
   );
 }
 
+function EmissionFactorTooltip({ t }: { t: TFunction<'translation', undefined> }) {
+  return (
+    <Portal.Root className="pointer-events-none absolute left-0 top-0 z-50 flex h-full w-full flex-col content-center items-center justify-center bg-black/20 pb-40">
+      <div className="dark:border-1 relative mx-6 h-auto min-w-64 rounded-xl border bg-zinc-50 p-4 text-left text-sm opacity-80 shadow-md dark:border-gray-700 dark:bg-gray-900">
+        {t('country-panel.emissionFactorDataSourcesTooltip')}
+      </div>
+      <button className="p-auto pointer-events-auto mt-2 flex h-10 w-10 items-center justify-center self-center rounded-full border bg-zinc-50 text-black shadow-md sm:hidden">
+        <HiXMark size="24" />
+      </button>
```

Comment:
This button is currently hidden on sizes where it should still be visible - perhaps we can remove the `sm:hidden` here as the whole thing is only visible on smaller screens? :) 

![Screenshot 2024-06-18 at 21 34 43](https://github.com/electricitymaps/electricitymaps-contrib/assets/3296643/ac1c0a85-614d-4ac4-8098-1e65e2a55878)